### PR TITLE
Added resource tags to recently added new templates.

### PR DESCRIPTION
**Pull request by tpinney**
This change incorporates the following two changes:

~~1. Removed the `Microsoft.Insights/webtests` resource from the `application-insights.json` template following a prior PR to move it into its own template file, `availability-tests.json`~~
2. Added `resourceTags` parameter to `dashboard.json`, `action-group.json` and `availability-test-alert.json` template files.

### CODE REVIEW

File: templates/application-insights.json [hunk 1]
```diff
@@ -12,13 +12,6 @@
       "defaultValue": "",
       "type": "string"
     },
-    "availabilityTests": {
```
Comment (rizzkhan1) [inline templates/application-insights.json]: Can we keep this availability test in here as well as the new file? All my templates are using this atm.

Comment (tpinney) [reply templates/application-insights.json]: Sure, didn't realise you were already using my availability test templates. Give me five to check in the code to restore it.

**Comment by tpinney** [comment]
@rizzkhan1 I've modified the PR to remove item one from the change. Just includes the new `resourceTag` parameters now.

**Review by rizzkhan1 — APPROVED** [review]
LGTM

**Pull request merged**